models/example/fct_order_items.sql

```diff
@@ -0,0 +1,38 @@
+{# 
+create a fct_order_items table joining orders to order items. 
+select all fields from order items and join in the order_date, 
+email and customer_id from the orders table
+#}
+with orderitems as (
+
+    select * from {{ ref('order_items_upload') }}
+
+),
+
+orders as (
+
+    select * from {{ ref('orders_upload') }}
+
+),
+
+final as (
+
+    select
+        orderitems.id as order_item_id,
+        orderitems.order_id,
+        orderitems.price,
```

Comment:
it is not typical to list out explicit order_item fields since you have already done that in the stg model. typically, it will be `select orderitems.*, orders.whichever fields`. let me know if this makes sense!